app/src/main/java/fr/free/nrw/commons/upload/ImageProcessingService.java

```diff
@@ -69,17 +71,16 @@ public ImageProcessingService(FileUtilsWrapper fileUtilsWrapper,
         Single<Integer> darkImage = checkDarkImage(filePath);
         Single<Integer> itemTitle = checkTitle ? validateItemTitle(uploadItem) : Single.just(ImageUtils.IMAGE_OK);
         Single<Integer> checkFBMD = checkFBMD(context,contentUri);
-
+        Single<Integer> checkEXIF = checkEXIF(filePath);
         Single<Integer> zipResult = Single.zip(duplicateImage, wrongGeoLocation, darkImage, itemTitle,
                 (duplicate, wrongGeo, dark, title) -> {
                     Timber.d("Result for duplicate: %d, geo: %d, dark: %d, title: %d", duplicate, wrongGeo, dark, title);
                     return duplicate | wrongGeo | dark | title;
                 });
-
-        return Single.zip(zipResult, checkFBMD, (zip, fbmd) -> {
-            Timber.d("zip:" + zip + "fbmd:" + fbmd);
-            return zip | fbmd;
-        });
+        return Single.zip(zipResult,checkFBMD,checkEXIF, (zip,fbmd,exif)->{
+            Timber.d("zip:"+zip+"fbmd:"+fbmd+"exif:"+exif);
+            return zip| fbmd| exif;
```

Comment:
Better put a space before `|`.
For this kind of conventions a good read is https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Reply:
Sure thanks for the resource @nicolas-raoul  :)